.github/workflows/beta-deployment.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     name: Deploy Firebase Project
     uses: StanfordBDHG/.github/.github/workflows/firebase-deploy.yml@v2
     with:
-        customcommand: "npm install --prefix functions --save-dev"
+        customcommand: "npm install --prefix models --save-dev && npm install --prefix functions --save-dev"
```

Comment:
Could we create a small npm run command on the root that defines this at a single place across different parts of GitHub Actions & README instructions?

Reply:
will do